pages/index.js

```diff
@@ -1,24 +1,47 @@
+import { useRouter } from 'next/router'
+import Select from 'react-select'
 import { useAuth } from 'use-auth0-hooks'
 
-import UserList from '../components/UserList'
+import ErrorEventsDashboard from '../components/ErrorEventsDashboard'
+import RequestLogsDashboard from '../components/RequestLogsDashboard'
 import { AUTH0_SCOPE } from '../util/constants'
 
 export default function Index () {
   const { isAuthenticated, isLoading } = useAuth({
     audience: process.env.AUTH0_AUDIENCE,
     scope: AUTH0_SCOPE
   })
+  const { push, query: { dashboard } } = useRouter()
   if (!isLoading && !isAuthenticated) {
     return (
       <div>
         Please log in to view the Admin Dashboard.
       </div>
     )
   }
+  const dashboardOptions = [
+    // TODO: Remove Home?
+    // TODO: Factor shared code with manage.js?
+    {label: 'Home'}, // value is undefined to match missing query param
+    {value: 'errors', label: 'Errors'},
+    {value: 'requests', label: 'Request logs'}
+  ]
   return (
     <div>
-      <h1>OTP Admin Dashboard Overview</h1>
-      <UserList />
+      <h1>Dashboard</h1>
+      <Select
```

Comment:
I'd much rather have tabs or some kind of navbar instead of a dropdown.

Reply:
See #7